client/src/models/execution_layer/model_registry.py

```
@@ -0,0 +1,656 @@
+from common.config import GasStrategy
+from common.constants import MODELS_FOLDER
+from common.contract_constants import ContractVerificationStrategy
+from common.eth import EthereumClient
+from common.logging import get_logger
+from models.execution_layer.base_input import BaseInput
+from models.execution_layer.model_config import (
+    VERIFICATION_STRATEGY_MAP,
+    ModelMetadata,
+    load_model_metadata,
+)
+from models.execution_layer.request_type import RequestType
```

Flake8 (7.2.0)

[error] 'models.execution_layer.request_type.RequestType' imported but unused

(F401)